src/utils/Interfaces.ts

```diff
@@ -0,0 +1,26 @@
+export interface BorrowEligibilityResp {
+  result: {
+    borrowingOptions: {
+      mobileNumber: string;
+      borrowingAmounts: {
+        id: number;
+        amount: number;
+        interest: number;
+      }[];
+    }[];
+  };
+}
+
+export interface bundlesResp {
```

Comment:
I think it's a good practice to start interface names with Capital letters, sometimes even with an `I`, depending on the rule or programming language

Reply:
Cool. I've updated it.